### PR TITLE
 feat(t04): search job by title

### PR DESCRIPTION
**Pull request by vutiendanhvgu01**
t04deploy.surge.sh

### CODE REVIEW

**Comment by Tiencmfaba** [comment]
why do you use index.html ?

File: src/containers/Home/index.tsx [hunk 1]
```diff
@@ -46,13 +63,71 @@ function Home() {
...
+    target: { value: React.SetStateAction<string> };
+  }) => {
+    setSearchInput(e.target.value);
+    setCurrentPage(1);
+  };
+  const handleSearch = () => {
+    dispatch(filterAction(searchInput));
+    setCurrentPage(1);
+  };
+  const handleEnterSearch = (e: any) => {
```
Comment (Tiencmfaba) [inline src/containers/Home/index.tsx:78]: don't use any at here

File: src/containers/Home/index.tsx [hunk 2]
```diff
@@ -46,13 +63,71 @@ function Home() {
...
+      <div className="header">
+        <img
+          src="https://itviec.com/assets/logo-itviec-4492a2f2577a15a0a1d55444c21c0fa55810822b3b189fc689b450fb62ce0b5b.png"
+          className="header__logo"
+          alt=""
+        />
+      </div>
+      <div className="search">
+        <div className="search__content">
+          <div className="job--available">
```
Comment (Tiencmfaba) [inline src/containers/Home/index.tsx:97]: check BEm

File: src/containers/Home/index.tsx [hunk 3]
```diff
@@ -46,13 +63,71 @@ function Home() {
...
+            <h2>{count} Có Việc Làm IT Cho Developer "Chất"</h2>
+          </div>
+          <div className="search__bar">
+            <div className="search--icon">
```
Comment (Tiencmfaba) [inline src/containers/Home/index.tsx:101]: search-bar__icon

Comment (vutiendanhvgu01) [reply src/containers/Home/index.tsx:101]: I have fixed

File: src/containers/Home/index.tsx (hunk 2 above)
Comment (vutiendanhvgu01) [reply src/containers/Home/index.tsx:97]: I have fixed

File: src/containers/Home/index.tsx (hunk 1 above)
Comment (vutiendanhvgu01) [reply src/containers/Home/index.tsx:78]: I have fixed

**Pull request merged**